libnavigation-core/src/main/java/com/mapbox/navigation/core/reroute/InternalRerouteController.kt

```diff
@@ -0,0 +1,3 @@
+package com.mapbox.navigation.core.reroute
+
+internal interface InternalRerouteController : NavigationRerouteController
```

Comment:
why do you need this interface?

Reply:
It's for the future so that I can break it. Since I already know that I need to break it in the follow-up PR, I decided to introduce it here (to remove this boiler-plate code from the PR with route indices logic).